apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/edit.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { IconButton, ServerSideRender, Toolbar } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { BlockControls } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+class SiteLogoEdit extends Component {
+	navigateToCustomerSiteIdentity() {
+		const siteIdenityLink = addQueryArgs( 'customize.php', {
+			'autofocus[section]': 'title_tagline',
+			return: window.location.href,
+		} );
+		window.location.href = siteIdenityLink;
+	}
+
+	render() {
+		return (
+			<Fragment>
+				<BlockControls>
+					<Toolbar>
+						<IconButton
+							icon="edit"
+							label={ __( 'Edit Site Logo' ) }
+							onClick={ this.navigateToCustomerSiteIdentity }
```

Comment:
Since we only want to redirect users to the customizer, and provided the `IconButton` component relies on `Button` which supports [links](https://github.com/WordPress/gutenberg/blob/a57202e635ff0cd190a09128d27951620db9a870/packages/components/src/button/index.js#L13), we might want to use a `href` property here rather than `onClick` which should be better in terms on accessibility.